client/verta/verta/registry/entities/_model.py

```diff
@@ -87,6 +90,11 @@ def workspace(self):
         else:
             return self._conn._OSS_DEFAULT_WORKSPACE
 
+    @property
+    def pii(self):
+        self._refresh_cache()
+        return self._msg.pii
```

Comment:
I'd say we should omit this; `get_pii()` should suffice, and it's not greeeaat to have multiple ways to do something simple.

I don't have a clear guideline for when something should or shouldn't be a `property`, but it's something along the lines of "`property` is for fundamental things about the entity that the user can't directly change"